include/dgl/nodeflow.h

```diff
@@ -42,6 +43,31 @@ struct NodeFlow {
    * \brief The edge mapping from the NodeFlow graph to the parent graph.
    */
   IdArray edge_mapping;
+
+  static constexpr const char* _type_key = "graph.NodeFlow";
+  DGL_DECLARE_OBJECT_TYPE_INFO(NodeFlowObject, runtime::Object);
+};
+
+class NodeFlow : public runtime::ObjectRef {
```

Comment:
Seems we can use macro to simplify this with macro ([code from tvm](https://github.com/dmlc/tvm/blob/415a270def327c4e1beeea75ce7f296e4683a5c0/include/tvm/base.h#L48)

Reply:
Great find.

Reply:
Done with the macro.